packages/core/src/api/__tests__/trimScale.ts

```diff
@@ -0,0 +1,20 @@
+import { calculator } from '@dinero.js/calculator-number';
```

Comment:
There's already a [test suite](https://github.com/dinerojs/dinero.js/blob/main/packages/dinero.js/src/api/__tests__/trimScale.test.ts) for `trimScale`, could you move the tests there?

Reply:
I did not find that at all, thanks! will do :)